handcrafts.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Products Landing Page</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="container">
+        <div class = "menu">
+            <div class="logo">
+                <img src="image/product-landing-page-logo.png" width="300px" alt="logo">
+            </div>            
+            <div class="menu_items">
+                <a href="#">Features</a></li>
+                <a href="#">How It Works</a>
+                <a href="#">Pricing</a>
+            </div>
```

Comment:
Semantic tags provide additional information to search engine and screen reader by providing context. So please use `<nav>` tag for the menu.